metricbeat/module/ceph/_meta/Dockerfile

```diff
@@ -3,5 +3,8 @@ FROM ceph/demo:tag-build-master-jewel-centos-7
 ENV MON_IP 0.0.0.0
 ENV CEPH_PUBLIC_NETWORK 0.0.0.0/0
 
-HEALTHCHECK --interval=1s --retries=90 CMD ceph --status | grep HEALTH_OK
+RUN yum -q install -y jq && yum clean all && rm -fr /var/cache/yum
+
+# Wait for the health endpoint to have monitors information
+HEALTHCHECK --interval=20s --retries=10 CMD curl -H "Accept: application/json" localhost:5000/api/v0.1/health | jq -e .output.health.health_services[0].mons[0] > /dev/null
```

Comment:
The compose timeout is 300s but here we have an interval of 20s and 10 retries which only sums up to 200s? 

In general as this is for testing purpose I would set the interval to 1s and retries to 300 in this case to have a container healthy as soon as possible as it will speed up tests.

Reply:
Ok, I changed this to avoid flooding the logs with the healthcheck requests, but yes, they are tests, I'll change this to your proposal.

Reply:
I'm going to reduce it also from 300 to 180, it should be enough, it gets healthy quite consistently after a minute.